dept44-service-parent/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>se.sundsvall.dept44</groupId>
```

Comment:
Remove version from git-commit-id-maven-plugin on row 342? Not necessary since the version is defined in root-pom.